### PR TITLE
Last node comparing metric

### PR DESCRIPTION
**Pull request by traeuker**
This is the first of a few behavioral statistics evaluation metrics. This metric provides insights into emerging capabilities by measuring the proximity of the predicted target to the actual target.

My first PR in a serious repo. Let me know if I messed anything up :')

### CODE REVIEW

**Comment by mivanit** [comment]
you should run `make format` and then commit to make sure the linting tests pass. otherwise, this looks good!

File: maze_transformer/evaluation/path_evals.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def fraction_connections_adjacent_lattice(prediction: MazePath, **_) -> float:
     @register_method(evals)
     @staticmethod
     def num_connections_adjacent(maze: LatticeMaze, prediction: MazePath, **_) -> float:
-        """number of connections in prediction which are are valid paths on the maze"""
```
Comment (luciaquirke) [inline maze_transformer/evaluation/path_evals.py:81]: doing the lord's work

**Review by luciaquirke — APPROVED** [review]
Nice one!

**Review by mivanit — CHANGES_REQUESTED** [review]
some minor issues with `corner_jumps`, otherwise LGTM!

**Review by mivanit — APPROVED** [review]

**Pull request merged**